packages/embed/package.json

```diff
@@ -30,7 +30,7 @@
     "@sentry/browser": "^8.33.0",
     "@sentry/esbuild-plugin": "^2.22.3",
     "@smile_identity/smart-camera-web": "file:../smart-camera-web",
-    "@smileid/web-components": "<2.0.0",
+    "@smileid/web-components": "file:../web-components",
```

Comment:
Are you sure about this change?

Reply:
This is necessary to deploy embed without always waiting for the web-component release first.
We just use the file dependency just like we do `"file:../smart-camera-web"`